### ISSUE TITLE
Popup sizing on firefox

### ISSUE THREAD

**Issue by canislupaster** [Thomas-QM/BookmarkSearch#1]
it sucks.

**Comment by grahamperrin** [comment]
Do you mean, when the icon is in the overflow menu?

**Comment by canislupaster** [comment]
nah, just its unnaturally big when displaying long results. instead of wrapping, it just resizes the popup. this doesn't happen on chrome.
